airflow/hooks/S3_hook.py

```diff
@@ -384,3 +384,89 @@ def load_bytes(self,
 
         client = self.get_conn()
         client.upload_fileobj(filelike_buffer, bucket_name, key, ExtraArgs=extra_args)
+
+    def _copy_object(self,
```

Comment:
I name the methods in `S3Hook()` as `_copy_object`, in order to avoid confusion between `boto3.client.copy_object` and `S3Hook.copy_object`.

The same applies to  `_delete_objects`

Reply:
I wouldn't. A method prefixed with `_` in python is usually an indication that it is "private" and shouldn't be called from outside the method.

Given the S3Hook doesn't directly expose any methods from the boto client/session object I think the chance of confusion is slight. (and a doc string highlighting any differences will help dispel confusion)

Reply:
Sure, I will change this part.

Actually the chance of misusing is low given the argument required are totally different.